src/components/ManageAssets/ManageAssetsDropdown/ManageAssetsDropdown.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import styled from "@emotion/styled";
+import { mixin, vars } from "../../../styles";
+import { Dropdown } from "react-bootstrap";
+import { loadAdminAddress } from "../../../reducers/admin";
+import { connectToMetamask } from "../../../services/etherjs";
+
+interface ManageAssetsDropdownProps {
+  userWalletAddress: string;
+  approvedBeneficiaryAddress: string;
+  approvedHolderAddress: string;
+  beneficiaryAddress: string;
+  holderAddress: string;
+  metamaskAccountError: string;
+  className?: string;
+}
+
+export const ManageAssetsDropdown = ({
+  userWalletAddress,
+  approvedBeneficiaryAddress,
+  approvedHolderAddress,
+  beneficiaryAddress,
+  holderAddress,
+  metamaskAccountError,
+  className,
+}: ManageAssetsDropdownProps) => {
+  const dispatch = useDispatch();
+
+  const isApprovedAddressPending = !!approvedBeneficiaryAddress && !!approvedHolderAddress;
+  const userNotLoginMetamask = !userWalletAddress && metamaskAccountError;
+  const userNoManagePrivileges =
+    (userWalletAddress !== holderAddress && userWalletAddress !== beneficiaryAddress) ||
+    (userWalletAddress !== holderAddress && userWalletAddress === beneficiaryAddress && isApprovedAddressPending);
+
+  const handleSelectedTransfer = (event: { target: any }) => {
+    console.log(event.target);
+  };
+
+  const handleNoAccess = () => {
+    alert("User has no manage asset privileges.");
+  };
+
+  const handleConnectMetamask = async (event: { preventDefault: () => void }) => {
+    event.preventDefault();
+    await connectToMetamask();
+    dispatch(loadAdminAddress());
+  };
+
+  return (
+    <Dropdown className={`${className}`} alignRight>
+      <Dropdown.Toggle
+        variant="primary"
+        id="dropdown-basic"
+        {...(userNotLoginMetamask && { onClick: handleConnectMetamask })}
+        {...(userNoManagePrivileges && { onClick: handleNoAccess })}
```

Comment:
hide the manage asset button if user doesn't have access, No need to show the button and for alert.

Reply:
it is supposed to have an overlay response as to the design, this will be done in a separate PR.